proto/ethermint/sgx/v1/query.proto

```diff
@@ -0,0 +1,396 @@
+  // *big.Int
+	uint64 homestead_block  = 2; // Homestead switch block (nil = no fork, 0 = already homestead)
+
+  // *big.Int
+	uint64 d_a_o_fork_block  = 3;  // TheDAO hard-fork switch block (nil = no fork)
+	bool d_a_o_fork_support = 4; // Whether the nodes supports or opposes the DAO hard-fork
+
+	// EIP150 implements the Gas price changes (https://github.com/ethereum/EIPs/issues/150)
+  // *big.Int
+	uint64 e_i_p_150_block = 5; // EIP150 HF block (nil = no fork)
+  // *big.Int
+	uint64 e_i_p155_block = 6; // EIP155 HF block
+  // *big.Int 
+	uint64 e_i_p158_block = 7; // EIP158 HF block
+
+  // *big.Int
+	uint64 byzantium_block = 8; // Byzantium switch block (nil = no fork, 0 = already on byzantium)
+  // *big.Int
+	uint64 constantinople_block = 9; // Constantinople switch block (nil = no fork, 0 = already activated)
+  // *big.Int 
+	uint64 petersburg_block = 10;     // Petersburg switch block (nil = same as Constantinople)
+  // *big.Int
+	uint64 istanbul_block = 11;       // Istanbul switch block (nil = no fork, 0 = already on istanbul)
+  // *big.Int
+	uint64 muir_glacier_block = 12;    // Eip-2384 (bomb delay) switch block (nil = no fork, 0 = already activated)
+  // *big.Int
+	uint64 berlin_block = 13;         // Berlin switch block (nil = no fork, 0 = already on berlin)
+  // *big.Int
+	uint64 london_block = 14;         // London switch block (nil = no fork, 0 = already on london)
+  // *big.Int
+	uint64 arrow_glacier_block = 15;   // Eip-4345 (bomb delay) switch block (nil = no fork, 0 = already activated)
+	// *big.Int
+  uint64 gray_glacier_block = 16;    // Eip-5133 (bomb delay) switch block (nil = no fork, 0 = already activated)
+	// *big.Int
+  uint64 merge_netsplit_block = 17;  // Virtual fork after The Merge to use as a network splitter
+
+	// Fork scheduling was switched from blocks to timestamps here
+
+  // *uint64
+	uint64 shanghai_time = 18[(gogoproto.nullable) = true]; // Shanghai switch time (nil = no fork, 0 = already on shanghai)
+  // *uint64
+	uint64 cancun_time   = 19[(gogoproto.nullable) = true];   // Cancun switch time (nil = no fork, 0 = already on cancun)
+  // *uint64
+	uint64 prague_time   = 20[(gogoproto.nullable) = true];   // Prague switch time (nil = no fork, 0 = already on prague)
+  // *uint64
+	uint64 verkle_time   = 21[(gogoproto.nullable) = true];   // Verkle switch time (nil = no fork, 0 = already on verkle)
+
+	// TerminalTotalDifficulty is the amount of total difficulty reached by
+	// the network that triggers the consensus upgrade.
+	// *big.Int 
+	uint64 terminal_total_difficulty = 22;
+
+	// TerminalTotalDifficultyPassed is a flag specifying that the network already
+	// passed the terminal total difficulty. Its purpose is to disable legacy sync
+	// even without having seen the TTD locally (safer long term).
+	bool terminal_total_difficulty_passed = 23;
+
+	// Various consensus engines
+	EthashConfig ethash = 24;
+	CliqueConfig clique = 25;
```

Comment:
Do we need this 2? We should try to put the minimum amount of data necessary. It seems these 2 are not needed